site/content/en/docs/user/kwokctl-snapshot/yaml-snapshot.demo

```diff
@@ -0,0 +1,32 @@
+# Let's get started with kwokctl!
+kwokctl create cluster
+
+# Use context.
+kubectl cluster-info --context kwok-kwok
```

Comment:
See this [comment](https://github.com/kubernetes-sigs/kwok/pull/1100/files#r1611268816).
Thanks for the clarification about switching context, @wzshiming.

```suggestion
```